chromadb/config.py

```diff
@@ -19,6 +19,7 @@ class Settings(BaseSettings):
     chroma_server_http_port: str = None
     chroma_server_ssl_enabled: bool = False
     chroma_server_grpc_port: str = None
+    chroma_server_cors_allow_origins: List[str] = ["http://localhost:3000"]
```

Comment:
maybe the default should be `[]` now that I think about it

Reply:
Yes, running with localhost in CORS by default could cause a security issue.